packages/react-jss/src/merge-classes.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type {Classes} from 'jss'
+
+/**
+ * This merges to classes object.
+ * When a property exists in both classes, it will not overwrite but merge the two classes.
+ *
+ * @param {Object} baseClasses - The base classes.
```

Comment:
we should drop jsdoc anotations since we use flow already, maintaining both is bad, because it will get out of sync without validation